packages/components/src/components/form/bal-input/bal-input-util.ts

```diff
@@ -72,6 +72,62 @@ export function formatOffer(value: string): string {
       return `${parts[0]}/${parts[1]}.${parts[2]}.${parts[3]}`
   }
 }
+
```

Comment:
@JasperDeLanghe could you add a unit test for those functions?

Reply:
@hirsch88 see [new commit](https://github.com/baloise-incubator/design-system/pull/1146/commits/0635cb0b4ae06bba94e05caac4ac28ce02405070) for unit tests